app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration67AndroidTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.simple.clinic.storage.migrations
+
+import org.junit.Test
+import org.simple.clinic.assertColumns
+
+class Migration67AndroidTest : BaseDatabaseMigrationTest(66, 67) {
+
+  @Test
+  fun deleted_reason_should_be_added_to_patient_table() {
```

Comment:
Can we also add another test to verify that the patient record is retained after the migration?